yb-voyager/src/srcdb/postgres.go

```diff
@@ -463,6 +463,17 @@ func (pg *PostgreSQL) GetCharset() (string, error) {
 	return encoding, nil
 }
 
+func (pg *PostgreSQL) GetDatabaseSize() (int64, error) {
+	var dbSize int64
+	query := fmt.Sprintf("select pg_database_size('%s'); ", pg.source.DBName)
```

Comment:
@priyanshi-yb in case of postgres, we also take input of schema-list from user.
So user is migrating a subset of schema then taking full db size is not correct.

Sharing some sample queries given by chatgpt for doing that. Please verify them before using.
1.
```
SELECT
    nspname AS schema_name,
    pg_size_pretty(SUM(pg_total_relation_size(pg_class.oid))) AS size
FROM
    pg_namespace
JOIN
    pg_class ON pg_class.relnamespace = pg_namespace.oid
WHERE
    nspname IN ('schema1', 'schema2', 'schema3') -- Add your schema names here
GROUP BY
    nspname
ORDER BY
    nspname;
```

2. 
```
WITH schema_tables AS (
    SELECT table_schema AS schema_name,
           table_name
    FROM information_schema.tables
    WHERE table_schema IN ('schema1', 'schema2', 'schema3') -- Add your schema names here
),
schema_sizes AS (
    SELECT schema_name,
           ROUND(SUM(pg_total_relation_size(schema_name || '.' || table_name)) / 1024 / 1024, 2) AS size_mb
    FROM schema_tables
    GROUP BY schema_name
)
SELECT schema_name, size_mb
FROM schema_sizes;

```

Reply:
I was thinking this `total_db_size` in source_db_details is the total database size of the pg for which migration is happening not only the schemas that are getting migrated because if we go with schemas specific then it is already present in assess-migration's table/index sizing stats in some way. So having this total_db_size as the total pg db size is fine. We can this one in standup with others as well and if required I can modify this later.

Reply:
Sure we can discuss this in standup.
But how is it of any use to consider for the schemas we are not migrating them.

Callhome user might draw wrong conclusions if some large schemas are present which are not migrated